### ISSUE TITLE
hover over a variable/property in code to expose the variable type via small tooltip box

### ISSUE THREAD

**Issue by the-packrat** [godotengine/godot-proposals#8138]
### Describe the project you are working on

this proposal sounds universally applicable in the realm of GDscript.

### Describe the problem or limitation you are having in your project

it's dizzying for those with attention deficit disorders to open dozens of documentation pages in order to confirm the type of variable/property they just accessed in their code. especially so if they happen to have a sinus infection.

### Describe the feature / enhancement and how it helps to overcome the problem or limitation

this **accessibility feature** would allow users with attention deficit disorders to stay on the page they are working on, while confirming via a small tooltip box the type of property or variable they accessed.

### Describe how your proposal will work, with code, pseudo-code, mock-ups, and/or diagrams

in practice, the users would hover over a variable or property, and a small box appears to display the type of variable/property after X number of milliseconds.
in short, the behavior of the box should match that of a "tooltip".

> The tooltip, also known as infotip or hint, is a common graphical user interface element in which, when hovering over a screen element or component, a text box displays information about that element,

-Wikipedia

### If this enhancement will not be used often, can it be worked around with a few lines of script?

within my small scope of expertise, no.

### Is there a reason why this should be core and not an add-on in the asset library?

in good execution, the usefulness to those who want the feature would far outweigh the possible clutter it would inflict on those who didn't ask for this.

**Comment by KoBeWi** [comment]
Thanks for the proposal! Consolidating in #1393
